src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -7,6 +7,33 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* Removed the following package references:
+
+  * `Google.Protobuf`
+  * `Grpc`
+  * `Grpc.Net.Client`
+  * `Grpc.Tools`
+
+  These changes were made to streamline dependencies and reduce the footprint of
+  the exporter.
+  ([#6005](https://github.com/open-telemetry/opentelemetry-dotnet/pull/6005))
+
+* Switched from using the `Google.Protobuf` library for serialization to a
+  custom manual implementation of protobuf serialization.
+  ([#6005](https://github.com/open-telemetry/opentelemetry-dotnet/pull/6005))
+
+* Fixed an issue where a `service.name` was added to the resource if it was
+  missing. The exporter now respects the resource data provided by the SDK
+  without modifications.
+  ([#6015](https://github.com/open-telemetry/opentelemetry-dotnet/pull/6015))
+
+* Removed the peer service resolver, which was based on earlier experimental
```

Comment:
Let's spin up an issue to completely remove the code for peer service resolver and update zipkin too.

Reply:
Created an issue to track it - https://github.com/open-telemetry/opentelemetry-dotnet/issues/6018